### ISSUE TITLE
normalization issue for `DTCWTDirect`

### ISSUE THREAD

**Issue by xir4n** [kymatio/murenn#47]
the coefficients don't seem to be normalized correctly when setting `alternate_gh=True`:
```python
N = 44100
x = torch.zeros(1, 1, N)
x[0, 0, N//2] = 1
J = 8

dtcwt = murenn.DTCWT(J=J, alternate_gh=False)
idtcwt = murenn.DTCWTInverse(J=J, alternate_gh=False)

x_phi, x_psis = dtcwt(x)
y_j = []

for j in range(J):
    y_phi = x_phi * 0
    y_psis = [x_psis[k] * (j==k) for k in range(J)]
    y_j.append(idtcwt(y_phi, y_psis).squeeze())

lp_psis = [torch.zeros(x_psis[k].shape) + 1j*torch.zeros(x_psis[k].shape) for k in range(J)]
y_lp = idtcwt(x_phi, lp_psis).squeeze()

plt.figure(figsize=(10, 3))
for j in range(J):
    y_jhat = torch.fft.fft(y_j[j])
    plt.semilogx(range(N), torch.abs(y_jhat), label=f'j={j+1}')
y_lphat = torch.fft.fft(y_lp)
plt.semilogx(range(N), torch.abs(y_lphat), label=f'lp')
plt.grid(linestyle='--', alpha=0.5)
plt.xlim(0.5, N//2)
plt.title('Frequency responses')
plt.legend()
``` 
output: 
![image](https://github.com/kymatio/murenn/assets/97965893/6b0a5224-3700-40de-b8e0-f94f081c9888)

**Comment by lostanlen** [comment]
after our conversation today, it seems like the best option is simply to remove the `normalize` optional argument?

**Comment by xir4n** [comment]
![Pasted image 20240722183000](https://github.com/user-attachments/assets/b279a12c-3d2f-4f1a-936a-ab678a95d79e)

**Comment by lostanlen** [comment]
seems like setting `alternate_gh=False` in https://github.com/kymatio/murenn/pull/56 solved the problem ... 
shall we close this?
